src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -154,7 +178,6 @@
                                       x:Uid="ImageResizer_Size_Property"
                                       SelectedIndex="{Binding Path=Unit, Mode=TwoWay}"
                                       Width="120"
-                                      Height="34"
```

Comment:
I'd go for MinHeight instead of removing it entirely to make sure it renders properly when setting display scaling to e.g. 100%/150%